src/server/apollo/dataSources.ts

```diff
@@ -48,11 +48,15 @@ const transformGraphQLVariables = (query: string, variables: any = {}) => {
       get(def, 'type.name.value') === 'JSON' &&
       get(variables, def.variable.name.value)
     ) {
-      set(
-        variables,
-        def.variable.name.value,
-        JSON.stringify(get(variables, def.variable.name.value))
-      );
+      let value = get(variables, def.variable.name.value);
+      if (value && typeof value === 'object') {
+        if (value.hasOwnProperty('AND') && value.AND !== null) {
```

Comment:
@GuilhermeGabriel 
that can hardly be considered as a valid solution
it is really really too specific to the use case we have